Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -39,6 +39,9 @@ public class ErrorPlugin : EventProcessorPluginBase {
 
             error.Data[Error.KnownDataKeys.TargetInfo] = targetInfo;
 
+            if (context.StackSignatureData.ContainsKey(nameof(Event.KnownDataKeys.ManualStackingKey)))
```

Comment:
I think we should really be just checking to see if StackSignatureData.Count > 0 and exiting. If stacking information has been supplied in some other way then we probably shouldn't override it.